lib/vol_av.h

```diff
@@ -3,7 +3,7 @@
  *
  * vol_av    | Audio-Video Decoding API
  * --------- | ----------
- * Version   | 0.7.1
+ * Version   | 0.9
```

Comment:
these files (vol_av.*, vol_geom.*) are libraries updated. ignore changes here - reviewed in vol_libs PRs.